### PR TITLE
Cratesio publish

### PR DESCRIPTION
**Pull request by Ropes**
rebased #597 to solve Dockerfile conflict. Opening PR to run CI.

Verification needed:
* [ ] kubos-dev docker image

## Original PR

This PR makes it much more straight forward to build a KubOS service out-of-tree by using built in Cargo dependency management and publishing some of Kubos' high level crates to crates.io.

Currently this PR is focused on only publishing these crates:

    kubos-app
    kubos-comms (formerly comms-service)
    kubos-system
    kubos-service
    kubos-telemetry-db

### CODE REVIEW

File: tools/dist/Dockerfile [hunk 1]
```diff
@@ -37,14 +37,14 @@ RUN apt-get install --no-install-recommends -y \
     unzip \
     wget
 
+
+#Kubos Linux setup
+RUN echo "Installing Kubos Linux Toolchain"
+
```
Comment (marshall) [inline tools/dist/Dockerfile:43]: remove this

**Review by marshall — CHANGES_REQUESTED** [review]
it looks like the Dockerfile got a bit reverted in the rebase, I think because this was based on an older version. I marked the differences we need in the Dockerfile

File: tools/dist/Dockerfile [hunk 2]
```diff
@@ -37,14 +37,14 @@ RUN apt-get install --no-install-recommends -y \
...
 # Custom gcc toolchains
-RUN wget https://s3.amazonaws.com/kubos-world-readable-assets/iobc_toolchain.tar.gz && \
-    tar -xf ./iobc_toolchain.tar.gz -C /usr/bin && \
-    rm ./iobc_toolchain.tar.gz
+RUN wget https://s3.amazonaws.com/kubos-world-readable-assets/iobc_toolchain.tar.gz && tar -xf ./iobc_toolchain.tar.gz -C /usr/bin && rm ./iobc_toolchain.tar.gz
```
[the remark this answers is not in the document]
Comment (Ropes) [reply tools/dist/Dockerfile:45]: Got it.

**Comment by Ropes** [comment]
I think we should decide yay or nay on these changes. We've both run into issues compiling that Cross hasn't solved, but the configuration updates are still steps forward.(I think)

**Review by marshall — APPROVED** [review]
sorry for letting this fester!

**Pull request merged**